### DISCUSSION TITLE
求助大佬，现在复制svg去sketch里，没有正方形的透明背景了嘛？

### DISCUSSION THREAD

**Discussion by wenzis666** [Remix-Design/RemixIcon#532] [General]
感谢大佬

**Comment by xiaochunjimmy** [comment]
事情是这样的：

- 越来越多的设计师使用 Figma 替代 Sketch（也包括我），在 Figma 它会保留图标的尺寸，及时没有那个透明背景，也能撑满；反而透明背景会让图层不干净；
- 相反在 Sketch 里面默认是一个组，必须用一个透明的图层来撑开才能保证统一大小；
- 而最近的 3.0 版本，我们改用了 Figma 来管理图标，并自动化了图标更新流程（发布新版本的时候 API 自动从设计稿中获取 SVG 并发布到网站），这导致原本的透明背景不见了，也导致 Sketch 用户很难受，这两天也收到了几个反馈，让我意识到了问题的严重性，我们的锅...
- 所以计划做成是一个勾选项，用户可以在复制 Svg 的时候自由选择，同时兼顾 Sketch 用户和 Figma 用户；

最后，一点点小请求，麻烦在 GitHub 右上角点个 Star，万分感谢，Star 主要用来标记自己感兴趣的项目作为收藏，同时也代表项目的欢迎程度

**Comment by xiaochunjimmy** [comment]
网站更新后新增了功能来满足你的需求，你看下是否能快速找到这个功能，顺便帮我们做个小用户测试。我就不详细展开了了。有问题随时回这里来沟通

**Reply by wenzis666** [reply]
感谢大佬无私奉献！！已点star～
